ride-service/src/main/java/com/modsen/ride/constants/ControllerMappings.java

```diff
@@ -0,0 +1,6 @@
+package com.modsen.ride.constants;
+
+public interface ControllerMappings {
```

Comment:
Использовать интерфейс для констант не очень хороший вариант, лучше используй класс (можно использовать для класса ломбок аннотацию `@UtilityClass`):
```suggestion
public final class ControllerMappings {
    public static final String RIDE_CONTROLLER = "/rides";
    public static final String RIDE_OPERATIONS_CONTROLLER = "/ride-operations";
    
    private ControllerMappings(){}
}
```
Посмотри и замени все остальные интерфейсы, используемые для констант